pkgs/top-level/all-packages.nix

```diff
@@ -9034,7 +9034,6 @@ in
   } // (stdenv.lib.optionalAttrs (stdenv.cc.isGNU && stdenv.hostPlatform.isi686) {
       stdenv = gcc6Stdenv; # with gcc-7: undefined reference to `__divmoddi4'
   }));
-  spidermonkey_52 = callPackage ../development/interpreters/spidermonkey/52.nix { };
```

Comment:
Do we want aliases that throw to use the latest attribute?

Reply:
I guess, no idea if this was a common package for users to install/use.